jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2643,7 +2643,7 @@ boolean isPrimaryType() {
 }
 boolean isPrimitiveValueType() {
 	// ToDo: this is a temporary implementation - https://github.com/eclipse-openj9/openj9/issues/13615
-	return false;
+	return isPrimitiveClass();
```

Comment:
If this is the permanent solution, the `ToDo` comment should be removed.

Reply:
This is a temporary solution.